diff --git a/node_modules/expo-audio/ios/AudioUtils.swift b/node_modules/expo-audio/ios/AudioUtils.swift
index 66916fb..a35daaa 100644
--- a/node_modules/expo-audio/ios/AudioUtils.swift
+++ b/node_modules/expo-audio/ios/AudioUtils.swift
@@ -108,7 +108,7 @@ struct AudioUtils {
     }
 
     let asset = AVURLAsset(url: finalUrl, options: options)
-    return AVPlayerItem(asset: asset, automaticallyLoadedAssetKeys: [.tracks, .duration])
+    return AVPlayerItem(asset: asset, automaticallyLoadedAssetKeys: [.tracks])
   }
 
   static func createRecordingOptions(_ options: RecordingOptions) -> [String: Any] {
